.ebextensions/02_sidekiq_hooks.config

```
@@ -0,0 +1,49 @@
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/03_mute_sidekiq.sh":
```

```suggestion
  "/opt/elasticbeanstalk/hooks/appdeploy/pre/03_mute_sidekiq":
```
sacale la extension a todos los archivos